app/src/main/java/org/simple/clinic/sync/DataSync.kt

```diff
@@ -48,7 +48,7 @@ class DataSync(
 
   @Inject
   constructor(
-      modelSyncs: ArrayList<ModelSync>,
+      modelSyncs: List<@JvmSuppressWildcards ModelSync>,
```

Comment:
When do we need `JvmSuppressWildcards` ?

Reply:
Dagger has an issue with injecting wildcard generics (If you look at `SyncModule`, it provides `frequentlySyncingRepositories` as `List<SynceableRepositories<*, *>>` which are wildcards. This is a problem only when you use Kotlin, which has this annotation that you can use to fix this problem.